libsplinter/src/biome/user/store/mod.rs

```diff
@@ -33,6 +33,7 @@ impl SplinterUser {
     ///
     /// * `user_id`: unique identifier for the user being created
     ///
+    #[allow(dead_code)]
```

Comment:
This implies that something is not correctly guarded by a feature.

Reply:
Changed to rest-api guard